specification/apimanagement/resource-manager/Microsoft.ApiManagement/preview/2020-06-01-preview/definitions.json

```diff
@@ -5901,6 +5901,112 @@
           "description": "UTC Timestamp when the service was soft-deleted. The date conforms to the following format: yyyy-MM-ddTHH:mm:ssZ as specified by the ISO 8601 standard."
         }
       }
+    },
+    "PortalRevisionCollection": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PortalRevisionContract"
+          },
+          "description": "Collection of portal revisions.",
+          "readOnly": true
+        },
+        "count": {
```

Comment:
There is no count property in the pagination contract.  See https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/resource-api-reference.md#response-3

Reply:
+1